keps/sig-network/2030-endpointslice-subsetting/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: EndpointSlice Subsetting and Selection
+kep-number: 2030
+authors:
+  - "@robscott"
+  - "@jeremyot"
+owning-sig: sig-network
+participating-sigs:
+  - sig-multicluster
+status: implementable
+creation-date: 2020-09-29
+reviewers:
+  - "@andrewsykim"
+  - "@bowei"
+  - "@danwinship"
+  - "@thockin"
```

Comment:
Can you please add me as a reviewer?
[And I guess sig-scalability as participating sig]

Reply:
Added both, thanks!